terra/reports.py

```diff
@@ -734,3 +730,12 @@ def employee_total_report(employee_ids, start_date, end_date):
         employee_totals[e] = employee
 
     return employee_totals
+
+
+def fiscal_year_list():
```

Comment:
I suggest you put this function in the `utils.py` file with the other fiscal year functions. And I suggest your put the inception date itself in the `settings.py` file.

Reply:
@joshuago78 , made the changes suggested above.